nginx/tests/utils.py

```diff
@@ -18,39 +18,42 @@ def mocked_perform_request(*args, **kwargs):
     response = mock.MagicMock()
     url = args[0]
 
-    if '/2/nginx' in url:
+    if '/2/nginx' in url or '/3/nginx' in url:
```

Comment:
Let's try not to duplicate all endpoints.
Either we can loosen the condition with `if '/nginx' in url` or we use an inlined regex like so
```python
if re.search('/\d/nginx', url):
```

Reply:
Yeah, though I'd suggest to specifically declare the numbers

`if re.search('/[23]/nginx',url):`

It would make the versions supported by the tests clear and clearly shows the path forward when an api version's payload differs.